__tests__/helpers/buildSortQuery.test.js

```diff
@@ -0,0 +1,25 @@
+const { buildSortQuery } = require("../../helpers/buildSortQuery");
+
+describe("buildSortQuery function", () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  const fakeQuery = {
+    a: "1",
+    b: "-1"
+  };
+
+  it("should return query for sorting, when we passed sorting string", () => {
+    const fakedSortQuery = jest
+      .fn(Object.fromEntries)
+      .mockImplementationOnce(fakeQuery);
+
+    const result = buildSortQuery("asc(title),desc(_id)");
+    console.log("result:", result);
+
+    expect(fakedSortQuery).toHaveBeenCalledWith("asc(title),desc(_id)");
+
+    expect(result).toBe(fakeQuery);
+  });
+});
```

Comment:
const { buildSortQuery } = require("../../helpers/buildSortQuery");

describe("buildSortQuery function", () => {

  it("should return query for sorting, when we passed sorting string", () => {

   const params = "asc(title),desc(_id)";
   const expected = {title: -1, _id: 1}

    const result = buildSortQuery(params);
    expect(JSON.stringify(result)).(JSON.stringify(expected));
  });
});